ruby_on_rails/app_initialisation.md

```diff
@@ -19,14 +15,21 @@ You can do this with `gem update rails`. Beware of beta versions.
 You may want to choose a different database from Postgres, but most of the time that will be your choice.
 If you do not need a DB you may rethink the fact that you may not need Rails at all :) Take a look at [Sinatra](http://www.sinatrarb.com/) or [Angular](https://angular.io/)
 
+* Switch your local Ruby to the fresh one: `asdf local ruby latest` to populate a `.tool-versions` file
+  and delete any `.ruby-version` or `.nvmrc` files generated by Rails.
+
 * Load the Ruby version automatically in the fresh project's `Gemfile` by adding this:
 
-  ```ruby File.read(File.join(__dir__, '.ruby-version'))```
+  ```rb
+  ruby File.read(File.join(__dir__, '.tool-versions'))[/ruby\s(.*)/, 1].strip
```

Comment:
I thought about completely dropping this and replacing it by

```rb
ruby '~> 3.1.0'
```

since patch level updates never were a problem since Ruby 1.9.3. But I then thought, that we always want to go through the CI on a version change. So I now read the more complicated `.tool-versions` file.

Reply:
If Heroku and Semaphore are ok, I am ok 👍

Reply:
FYI, There'd also be a way to configure asdf to use the `.ruby-version` file:
https://github.com/asdf-vm/asdf-ruby#migrating-from-another-ruby-version-manager
Still using `.ruby-version` maybe is easier since Heroku etc. already know it (maybe they also look at .tool-versions, I haven't checked 🤷)

Reply:
@lukasbischof the assumption that Heroku knows about it is wrong. Heroku looks at the Gemfile.